packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -81,13 +77,12 @@ const convertRemoteParticipantsToParticipantListParticipants = (
             participant.role,
             isHideAttendeeNamesEnabled
           );
+          let remoteParticipantReaction = undefined;
```

Comment:
Why do we need to assign it to another 2 variables? Looks like remoteParticipantReaction is only used by Reaction? And spotlight is only used by spotlight?

Reply:
I explain why [here](https://github.com/Azure/communication-ui-library/pull/4415#discussion_r1552748646)